### PR TITLE
Fix overwriting pvc issue in mock mode by removing '-' prefix

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1060#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mphammer** [comment]
I will fix this in a separate PR for 11.x and 12.x

File: pkg/synopsysctl/cmd_create.go [hunk 1]
```diff
@@ -317,15 +317,15 @@ var createBlackDuckCmd = &cobra.Command{
...
+			defaultPvcComponentsList, err := app.Blackduck().GetComponents(&blackduckv1.Blackduck{Spec: blackDuckSpec}, blackduckapp.PVCResources)
 			if err != nil {
 				return err
 			}
 			defaultPvcList := []blackduckv1.PVC{}
 			for _, defaultPvcComponent := range defaultPvcComponentsList.PersistentVolumeClaims {
 				defaultPvcComponentResourceQuantitySize := defaultPvcComponent.Spec.Resources.Requests[v1.ResourceStorage]
 				pvc := blackduckv1.PVC{
-					Name: defaultPvcComponent.Name,
+					Name: defaultPvcComponent.Name[1:],
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_create.go]: Please add a unit test to avoid regression 😃

Comment (mphammer) [reply pkg/synopsysctl/cmd_create.go]: I'm going to add the Unit Tests in a different PR

**Pull request merged**